build.sh

```diff
@@ -162,13 +162,38 @@ perl .pre-process-annotate-attributes.pl < $HTML_TEMP/source-expanded-1 > $HTML_
 perl .pre-process-tag-omission.pl < $HTML_TEMP/source-expanded-2 > $HTML_TEMP/source-whatwg-complete # this one could be merged
 mkdir $HTML_TEMP/wattsi-output
 
+function rerunWattsi {
+  # If we end up here it means that wattsi exited non-zero, probably with
+  # one or more error messages that include line and column numbers; e.g.:
+  #     Parse Error:(748,59) unexpected end tag
+  # But in that case, the line numbers that wattsi reports are for lines in
+  # the copy of the source produced by running the original through the
+  # pre-processer steps above. Therefore, the reported line numbers may be
+  # wrong, so here we re-run wattsi against the original source to get the
+  # correct line numbers.
+  $QUIET || echo
+  $QUIET || echo "Line numbers in any error messages above may be wrong."
+  $QUIET || echo "The correct line numbers for errors in $HTML_SOURCE/source are shown below."
+  $QUIET || echo
+  mkdir $HTML_TEMP/parse2-output;
+  # XXX make this call to wattsi always be quiet after wattsi patch lands
+  wattsi $($QUIET && echo "--quiet") \
+    $HTML_SOURCE/source $HTML_TEMP/parse2-output \
+    $HTML_CACHE/caniuse.json $HTML_CACHE/w3cbugs.csv \
+    > $HTML_TEMP/parse.log || cat $HTML_TEMP/parse.log && exit 1
+    # unless this 2nd call to wattsi also exits non-zero, there aren't
```

Comment:
Should we cover that with a message? It seems a bit weird to say "correct line numbers are shown below" then show nothing.